frontend/packages/eslint-plugin-console/lib/config/typescript-parser.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
   overrides: {
     files: ['*.ts', '*.tsx'],
-    parser: 'typescript-eslint-parser',
-
-    // TODO update to new parser when https://github.com/prettier/prettier-eslint/issues/201 is fixed
```

Comment:
Do we need to bump any dependency to pick up this fix?

Reply:
I think we're safe. We're using `eslint-config-prettier` and not `prettier-eslint`. Perhaps I started with `prettier-eslint` and then moved on. Such a long time ago to remember :)
As long as everything is working in CI and IDE integration still.

Reply:
> Do we need to bump any dependency to pick up this fix?

For now, we can use the version of `@typescript-eslint/parser` that's already used in root `.eslintrc`.

This PR doesn't bump any dependency versions.

Reply:
> As long as everything is working in CI and IDE integration still.

For me, both `yarn lint` and VS Code IDE integration incl. Prettier still seem to work OK.